services/localfile/client/client.go

```diff
@@ -978,14 +978,15 @@ func (p *lsCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{
 }
 
 type cpCmd struct {
-	bucket    string
-	overwrite bool
-	uid       int
-	username  string
-	gid       int
-	group     string
-	mode      int
-	immutable bool
+	bucket          string
```

Comment:
Bucket is source file path. For a context it could be `file:///some-abs-path/`, `s3://s3-bucket/key/prefix/` and etc. It not influence on copy behavior

Reply:
I see, thanks